nimpy.nim

```diff
@@ -801,6 +804,32 @@ proc nimObjToPy[T](o: T): PPyObject =
     if ret != 0:
       cannotSerializeErr(k)
 
+proc nimJsonToPy(node: JsonNode): PPyObject =
+  case node.kind
+  of JNull:
+    result = newPyNone()
+  of JInt:
+    result = nimValueToPy(node.getInt)
+  of JFloat:
+    result = nimValueToPy(node.getFloat)
+  of JBool:
+    result = nimValueToPy(node.getBool)
+  of JString:
+    result = nimValueToPy(node.getStr)
+  of JArray:
+    result = pyLib.PyList_New(node.len)
+    for i in 0 ..< node.len:
+      let o = nimJsonToPy(node[i])
+      discard pyLib.PyList_SetItem(result, i, o)
```

Comment:
Should be a `decref o` here?

Reply:
Nope. I was confused when I saw your comment, but checked again. `PyList_SetItem` steals the reference. See the docs here:

https://docs.python.org/3.3/c-api/list.html?highlight=m#PyList_SetItem